binaries/conda/build_packages.py

```diff
@@ -1,23 +1,44 @@
-import os
-import sys
 import argparse
-import subprocess
+import os
+from datetime import date
 
 conda_build_dir = os.path.dirname(os.path.abspath(__file__))
 REPO_ROOT = os.path.join(conda_build_dir, "..", "..")
-MINICONDA_DOWNLOAD_URL = "https://repo.anaconda.com/miniconda/Miniconda3-py39_4.9.2-Linux-x86_64.sh"
-CONDA_BINARY = os.popen("which conda").read().strip() if os.system(f"conda --version") == 0 else  f"$HOME/miniconda/condabin/conda"
+MINICONDA_DOWNLOAD_URL = (
+    "https://repo.anaconda.com/miniconda/Miniconda3-py39_4.9.2-Linux-x86_64.sh"
+)
+CONDA_BINARY = (
+    os.popen("which conda").read().strip()
+    if os.system(f"conda --version") == 0
+    else f"$HOME/miniconda/condabin/conda"
+)
 
 if os.name == "nt":
-    #Assumes miniconda is installed in windows
+    # Assumes miniconda is installed in windows
     CONDA_BINARY = "conda"
 
+
+def add_nightly_suffix(ts_version, ma_version, wa_version):
```

Comment:
pypi and conda have different conventions for how we represent dates so let's have both functions and make it clear which one is conda and which one is for pypi

Reply:
Changed it